### ISSUE TITLE
Files in templates do not seem to be updated

### ISSUE THREAD

**Issue by grandeljay** [RobinTheHood/ModifiedModuleLoaderClient#160]
1. Create a module with files in /new_files/templates/tpl_modified/**
2. Publish the module, the template files will be copied to your active template (i. e. tpl_modified_responsive)
3. When you release a module update, the files in the active template do not seem to be updated

Please let me know if you need more details

**Comment by RobinTheHood** [comment]
That's correct. The problem is that many users adapt the template files to their design after installation. If an update is carried out, the MMLC would overwrite these adjustments. To be on the safe side, template files are not updated. You would have to come up with something for the MMLC if you wanted to automate this as well.

**Comment by grandeljay** [comment]
> The problem is that many users adapt the template files to their design after installation.

Even additional template files coming from a module?

This also affects PHP-Files in `/CURRENT_TEMPLATE/source/boxes`, which is breaking at least one of my modules after each update if I make changes there.

I understand that modified users change their template files but I doubt it also applies to template files from modules. 

It's natural that an update overwrites existing files, even modified itself is overwriting core and template files after an update. I believe it to be default and expected behaviour. Maybe the MMLC can do the same?

Alternately, maybe a flag in the config would be a workaround to tell the MMLC to overwrite all files during a module update.
